src/app/features/vendor/services/vendor.service.ts

```diff
@@ -12,38 +12,46 @@ export class VendorService {
 
   constructor(private http: HttpClient) {}
 
-  /**
+   /**
    * method to fetch markets from backend using GET method
    */
-  getMarkets(): Observable<IDropDownFields[]> {
+   getMarkets(): Observable<IDropDownFields[]> {
     return this.http.get<IDropDownFields[]>(`${this.baseUrl}/Market/GetMarkets`);
   }
-
+ 
   /**
    * method to fetch services from backend using GET method
    */
   getServices(): Observable<IDropDownFields[]> {
     return this.http.get<IDropDownFields[]>(`${this.baseUrl}/Service/GetService`);
   }
-
+ 
   /**
    * method to add  vendor  to backend using POST method
    */
   addVendor(vendorData: IVendorCreation): Observable<IVendorCreation> {
     return this.http.post<IVendorCreation>(`${this.baseUrl}/Vendor/CreateVendor`, vendorData);
   }
-
+ 
   /**
    * method to sending the updated vendor details to backend using PUT method
    */
   updateVendor(id: number,vendorData: IVendorCreation): Observable<IVendorCreation> {
     return this.http.post<IVendorCreation>(`${this.baseUrl}/Vendor/updateVendor/${id}`, vendorData);
   }
-
+ 
   /**
    * method to fetch the particular vendor details from backend
    */
-  getVendorById(id: number): Observable<IVendorData> {
-    return this.http.get<IVendorData>(`${this.baseUrl}/Vendor/GetVendorById/${id}`);
+  getVendorById(id: number): Observable<IVendor> {
+    return this.http.get<IVendor>(`${this.baseUrl}/Vendor/GetVendorById/${id}`);
   }
-}
\ No newline at end of file
+  
+  /**
+   * method to approve the vendor using PATCH method
```

Comment:
method description start with capital letter

Reply:
Changed that